server/web/api/register.py

```diff
@@ -1,27 +1,55 @@
 from functools import wraps
-import math
-from random import random, choice
-from flask import abort, make_response, request, Blueprint
+from flask import Response, abort, make_response, request, Blueprint
 from flask_api import status
 from flask_jwt_extended import create_access_token, get_jwt_identity, jwt_required
 from flask_login import login_user
 from flask_wtf.csrf import CSRFError, generate_csrf
+from werkzeug.exceptions import BadRequestKeyError
 from execution.entities.execution import Execution
-from models import WebUser
+from execution.services.entityloader import load_execution
+from execution.tests.entities.dummy_entities import create_test_execution
+import models
 from app_config import csrf
-from utils.tans import uniques
-from .test_data import test_execution
+from execution.run import activate_execution, active_executions
 
 # FIXME sollten wir zusammen mit dem Web package iwann mal umbenennen
 api = Blueprint("api-web", __name__)
 
 
+class ExecutionIdNotFound(Exception):
```

Comment:
If you want to create more of such exceptions, we might relocate them into an extra package. For now it is ok :).